.github/workflows/visual-tests.yml

```diff
@@ -0,0 +1,93 @@
+name: Visual Tests
+
+on: 
+  workflow_dispatch:
+    inputs:
+      update-snapshots:
+        description: "Update snapshots?"
+        type: boolean
+  
+
+ # Allow updating snapshots during manual runs
+  workflow_call:
+    inputs:
+      update-snapshots:
+        description: "Update snapshots?"
+        type: boolean
+
+jobs:
+  visual-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        project:
+          - name: Platform
+          - name: Install
+    env:
+      SHOPWARE_HTTP_CACHE_ENABLED: 0
+      SHOPWARE_DISABLE_UPDATE_CHECK: "true"
+      BLUE_GREEN_DEPLOYMENT: 1
+      COMPOSER_ROOT_VERSION: 6.7.9999999-dev
+    steps:
+      - name: Setup Shopware
+        uses: shopware/setup-shopware@main
+        with:
+          shopware-version: ${{ github.ref }}
+          shopware-repository: ${{ github.repository }}
+          install: true
+          install-admin: true
+          install-storefront: true
+          env: prod
+
+      - name: Start web server
+        run: symfony server:start -d
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install dependencies
+        working-directory: tests/acceptance
+        run: npm ci
+      - name: Install Playwright Browsers
+        working-directory: tests/acceptance
+        run: npx playwright install --with-deps chromium
+
+      - name: Download snapshot artifact
+        if: github.event_name == 'pull_request' || inputs.update-snapshots != 'true'
+        uses: actions/download-artifact@v4
+        with:
+          name: visual-snapshots
+          path: tests/screenshots
+        continue-on-error: true
+
+      - name: Run Playwright tests
+        run: |
+          if [ "${{ github.event.inputs.update-snapshots }}" = "true" ]; then
+            echo "Running with snapshot update"
+            npx playwright test --project=Visual --update-snapshots --reporter html
+          else
+            echo "Running without snapshot update"
+            npx npx pwc --project=Visual
+            continue-on-error: true
+            env:
+            CURRENTS_PROJECT_ID: ${{ secrets.CURRENTS_PROJECT_ID_VISUAL }}
+            CURRENTS_RECORD_KEY: ${{ secrets.CURRENTS_RECORD_KEY }}
```

Comment:
I don't quite understand the syntax here. Should these lines be passed to `npx` as arguments?